src/LoginOrApp.js

```diff
@@ -48,8 +48,11 @@ class LoginOrAppComponent extends Component {
     // split on /dashboard/
     // slashes are included in split so we do not also split on the second dashboard in /dashboard/dashboard
     const urlPostDashboard = window.location.href.split("/dashboard/")[1];
-    // if there was no /dashboard/ in the url, should only happen in dev or with a url of /dashboard (no tailing slash)
-    if (!urlPostDashboard) {
+    
+    if (
+      !urlPostDashboard ||  // // if there was no /dashboard/ in the url, should only happen in dev or with a url of /dashboard (no tailing slash)
+      urlPostDashboard[0] === "?" // or if the rest of the url starts with a question mark (?), this happens when the url has ?iframe=true as parameter
+    ) {
```

Comment:
Dit is een rijtje ifs, ik kan zo niet inschatten of dit voldoende is, en er is een kans dat het in de toekomst weer fouten introduceert. Is het niet beter om hierboven iets van `const urlPath = window.location.href.split('?')[0];` te doen en dan verder de oude logica te gebruiken? We waren er eerder niet van uit gegaan dat er een ? kon voorkomen.

Reply:
Ja terecht commentaar.
Ik heb het aangepast.
Zie ook de wijziging voor Parramatta bij:
https://github.com/nens/parramatta-dashboard/pull/147